lfs.c

```diff
@@ -4772,18 +4773,18 @@ int lfs_fs_traverse_(lfs_t *lfs,
         }
 
         if ((f->flags & LFS_F_DIRTY) && !(f->flags & LFS_F_INLINE)) {
-            int err = lfs_ctz_traverse(lfs, &f->cache, &lfs->rcache,
+            int err_traverse = lfs_ctz_traverse(lfs, &f->cache, &lfs->rcache,
                     f->ctz.head, f->ctz.size, cb, data);
-            if (err) {
+            if (err_traverse ) {
                 return err;
             }
         }
 
         if ((f->flags & LFS_F_WRITING) && !(f->flags & LFS_F_INLINE)) {
-            int err = lfs_ctz_traverse(lfs, &f->cache, &lfs->rcache,
+            int err_traverse  = lfs_ctz_traverse(lfs, &f->cache, &lfs->rcache,
                     f->block, f->pos, cb, data);
-            if (err) {
-                return err;
+            if (err_traverse ) {
+                return err_traverse ;
```

Comment:
Introduces white space/formatting change.

```suggestion
            if (err_traverse) {
                return err_traverse;
```